src/mono/wasm/runtime/icu.ts

```diff
@@ -30,8 +30,12 @@ export function init_globalization() {
     }
 
     const invariantEnv = "DOTNET_SYSTEM_GLOBALIZATION_INVARIANT";
+    const hybridEnv = "DOTNET_SYSTEM_GLOBALIZATION_HYBRID";
     const env_variables = runtimeHelpers.config.environmentVariables!;
-    if (env_variables[invariantEnv] === undefined && runtimeHelpers.invariantMode) {
+    if (env_variables[hybridEnv] === undefined && runtimeHelpers.config.globalizationMode === "hybrid") {
+        env_variables[hybridEnv] = "1";
+    }
+    else if (env_variables[invariantEnv] === undefined && runtimeHelpers.invariantMode) {
```

Comment:
True, at this stage in case we got them both set in the config, something must be wrong. It should throw.

Reply:
Now I can see: we are setting `runtimeHelpers.invariantMode` based on `runtimeHelpers.config.globalizationMode === "invariant"` value. That string cannot be simultaneously "invariant" and "hybrid", no threat here. It can stay as is.